Update copy-into-transact-sql.md (#9616)
MicrosoftDocs/sql-docs: docs/t-sql/statements/copy-into-transact-sql.md

   Select-AzSubscription -SubscriptionId <subscriptionId>
   Set-AzSqlServer -ResourceGroupName your-database-server-resourceGroup -ServerName your-SQL-servername -AssignIdentity
   ```

> ## Resolving HdfsBridge::recordReaderFillBuffer Errors

Reviewer: ```suggestion
## Resolve HdfsBridge::recordReaderFillBuffer errors

```

> You may experience the error HdfsBridge::recordReaderFillBuffer if types are mismatched between Parquet and SQL or if you have unsupported Parquet data types.
> Check this link to verify the that column names, order, and datatypes are correctly mapped: [Type Mapping for Parquet](https://learn.microsoft.com/en-us/azure/synapse-analytics/sql/develop-openrowset#type-mapping-for-parquet) 
> a quick way to resolve these or further troubleshoot is to load with AUTO_CREATE_TABLE = ON or create a table with all data types set to NVARCHAR. 

Reviewer 2: ```suggestion
You can experience the error `HdfsBridge::recordReaderFillBuffer` if types are mismatched between Parquet and [!INCLUDE[ssazuresynapse-md](../../includes/ssazuresynapse-md.md)], or if you have unsupported Parquet data types.

Verify that column names, order, and data types are correctly mapped, using the information in [Type mapping for Parquet](/azure/synapse-analytics/sql/develop-openrowset#type-mapping-for-parquet).

A quick way to resolve or to further troubleshoot this issue is to load with `AUTO_CREATE_TABLE = ON`, or create a table with all data types set to `NVARCHAR`. 
```

   Select-AzSubscription -SubscriptionId <subscriptionId>
   Set-AzSqlServer -ResourceGroupName your-database-server-resourceGroup -ServerName your-SQL-servername -AssignIdentity
   ```

> ## Resolve HdfsBridge::recordReaderFillBuffer errors

Reviewer: ```suggestion

## Resolve HdfsBridge::recordReaderFillBuffer errors
```
